src/Plugins/PublishResultModel.cs

```diff
@@ -20,6 +22,11 @@ public class PublishResultModel
         /// </summary>
         public string RootFileName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the root file name value.
```

Comment:
Is this the filename or the component name?

Reply:
So above, we already had RootFileName, that is the filename. I added RootComponentName, which will be the component name, same that shows in the emails

Reply:
OK so just fix the documentation where it says root file name value